dotnet/src/Connectors/Connectors.GoogleVertexAI.UnitTests/Core/StreamJsonParserTests.cs

```diff
@@ -121,7 +122,7 @@ public void ParseWhenStreamContainsOneObjectReturnsEnumerableWithOneObjectWithEs
         WriteToStream(stream, input);
 
         // Act
-        var result = parser.Parse(stream);
+        var result = parser.ParseAsync(stream);
```

Comment:
To test async, make sure your tests area async and you await the task. You may not be testing it this way, as they will run and the test will end before.

```csharp
public async Task ParseWhen...
...
var result = await parser.ParseAsync(stream);

```

Reply:
done